editor/store/reducer.js

```diff
@@ -317,6 +318,18 @@ export const editor = flow( [
 					},
 				};
 
+			case 'MOVE_BLOCK_TO_POSITION':
+				return {
+					...state,
+					[ action.uid ]: {
+						...state[ action.uid ],
+						attributes: {
+							...state[ action.uid ].attributes,
+							layout: action.layout,
```

Comment:
Should we be creating a whole new state value if `layout` hasn't changed for the block?